app/views/state_file/questions/md_primary_state_id/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<% title = t("state_file.questions.primary_state_id.edit.title") %>
+<% content_for :page_title, title %>
+
+<%# Must be above :card content_for %>
```

Comment:
hmm not sure why this comment is needed? what is the reason this needs to be above?
`content_for :card`

Reply:
For some reason it gets hidden if it's not above. The comment helps to provide that context.

Reply:
It's because the file executes in render order. If the content for this bit isn't provided by the time the partial is rendered, it will simply render without it.